.circleci/wait-for-job-finish.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Exit script if you try to use an uninitialized variable.
+set -o nounset
+
+# Use the error status of the first failure, rather than that of the last item in a pipeline.
+set -o pipefail
+
+counter=0
+
+# Wait up to 5 minutes for job to finish. A job can run on multiple nodes: parallelism > 1
```

Comment:
We don't know if the individual nodes will finish within 5 minutes of each; I've seen some pretty unbalanced times. I think we may need to increase this timeout, but we can monitor how it behaves in practice.